code/modules/mob/mob.dm

```diff
@@ -371,7 +371,7 @@
 		var/deathtimeseconds = round((deathtime - deathtimeminutes * 600) / 10,1)
 		usr << "You have been dead for[pluralcheck] [deathtimeseconds] seconds."
 
-		if (deathtime < 18000)
+		if (deathtime < 12000)
 			usr << "You must wait 30 minutes to respawn!"

```

Comment:
I didn't even notice that. I'll do a simple edit to 20 minutes, simply because I want to make as few edits as possible for merging's sake.